fedimint-client/src/sm/executor.rs

```diff
@@ -736,13 +736,20 @@ impl ExecutorInner {
 
 impl Debug for ExecutorInner {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        let (active, inactive) = tokio::runtime::Handle::current().block_on(async {
-            let active_states = self.get_active_states().await;
-            let inactive_states = self.get_inactive_states().await;
-            (active_states, inactive_states)
+        #[cfg(not(target_family = "wasm"))]
+        let (active, inactive) = fedimint_core::runtime::block_in_place(move || {
+            fedimint_core::runtime::block_on(async {
+                let active_states = self.get_active_states().await;
+                let inactive_states = self.get_inactive_states().await;
+
+                (active_states, inactive_states)
+            })
         });
```

Comment:
why are we even doing this, don't read database in debug impls

Reply:
Agreed.